sdk/containerregistry/container-registry/karma.conf.js

```diff
@@ -58,9 +58,9 @@ module.exports = function(config) {
       "TEST_MODE",
       "CONTAINER_REGISTRY_ENDPOINT",
       "CONTAINER_REGISTRY_ANONYMOUS_ENDPOINT",
-      "AZURE_CLIENT_ID",
-      "AZURE_CLIENT_SECRET",
-      "AZURE_TENANT_ID"
+      "CONTAINERREGISTRY_CLIENT_ID",
+      "CONTAINERREGISTRY_CLIENT_SECRET",
+      "CONTAINERREGISTRY_TENANT_ID"
```

Comment:
Plugin in the recorder to be updated to accept such variable name to be able to properly mock the login request.